bin/postman-test/collections/support-scheduler-importer.postman_collection.json

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
No intervalAction imported, so that "test 360" returned null body. Suggest posting the intervalAction from importer file.